pages/linux/ipscan.md

```diff
@@ -0,0 +1,29 @@
+# ipscan
+
+> A fast network scanner designed to be simple to use.
+> Also known as Angry IP Scanner.
+> More information: <https://angryip.org/>.
```

Comment:
```suggestion
> More information: <https://angryip.org/documentation/>.
```

Reply:
IMHO, the main page is easier to understand that the documentation, aka "longer essay". 😅